testplans/lotus-soup/testkit/role_miner.go

```diff
@@ -165,7 +165,7 @@ func PrepareMiner(t *TestEnvironment) (*LotusMiner, error) {
 			return nil, err
 		}
 
-		ds, err := lr.Datastore(context.Background(), "/metadata")
```

Comment:
It looks like this isn't using lotus from master - https://github.com/filecoin-project/lotus/blob/master/node/repo/interface.go#L58

Reply:
That's right, sorry about this.

The problem was that `build-lotus-soup` was missing a step to `go get github.com/filecoin-project/lotus@master` and this was indeed building an outdated commit from master.

I've added the step, so now on every CI run, we build against Lotus from master, which is what we want.